notebooks/curation_api/python/create_collection.ipynb

```diff
@@ -41,130 +41,89 @@
    "outputs": [],
    "source": [
-    "api_key = open(api_key_file).read().strip()  \n",
-    "access_token_headers = {\"x-api-key\": api_key}\n",
-    "access_token_path = \"/curation/v1/auth/token\"\n",
-    "access_token_url = f\"{api_url_base}{access_token_path}\"\n",
-    "res = requests.post(access_token_url, headers=access_token_headers)\n",
-    "access_token = res.json().get(\"access_token\")"
+    "collection_form_metadata = {\n",
+    "    \"name\": \"Name\",  # required\n",
+    "    \"description\": \"A sample description\",  # required\n",
+    "    \"contact_name\": \"First Last\",  # required\n",
+    "    \"contact_email\": \"firstlast@email.com\",  # required\n",
+    "    \"links\": [\n",
+    "        {\n",
+    "            \"link_name\": \"sample protocol link\",\n",
+    "            \"link_url\": \"https://www.sample.link\",\n",
+    "            \"link_type\": \"PROTOCOL\",\n",
+    "        },\n",
+    "        {\n",
+    "            \"link_name\": \"sample lab website\",\n",
+    "            \"link_url\": \"https://www.lab.website\",\n",
+    "            \"link_type\": \"LAB_WEBSITE\",\n",
+    "        }\n",
+    "    ],\n",
+    "}"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "e51b9169",
+   "id": "0009d2d0",
    "metadata": {},
    "source": [
-    "##### (optional, debug) verify status code of response"
+    "### Import dependencies"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "28978522",
+   "id": "aadd3c12",
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(res.status_code)"
+    "from src.utils.config import set_api_urls, set_access_token\n",
+    "from src.utils.logger import set_log_level  # can set_log_level(\"ERROR\") for less logging; default is \"INFO\"\n",
+    "from src.collection import create_collection"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "ce95a93a",
+   "id": "9501afeb",
    "metadata": {},
    "source": [
-    "### Collection metadata (required)"
+    "### Set url and access token env vars"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c711049d",
+   "id": "a6aab8e0",
    "metadata": {},
    "outputs": [],
    "source": [
-    "collection_form_metadata = {\n",
-    "    \"name\": \"Name\",\n",
-    "    \"description\": \"A sample description\",\n",
-    "    \"contact_name\": \"First Last\",\n",
-    "    \"contact_email\": \"firstlast@email.com\",\n",
-    "    \"links\": [\n",
-    "        {\n",
-    "            \"link_name\": \"sample protocol link\",\n",
-    "            \"link_url\": \"https://www.sample.link\",\n",
-    "            \"link_type\": \"PROTOCOL\",\n",
-    "        },\n",
-    "        {\n",
-    "            \"link_name\": \"sample lab website\",\n",
-    "            \"link_url\": \"https://www.lab.website\",\n",
-    "            \"link_type\": \"LAB_WEBSITE\",\n",
-    "        }\n",
-    "    ],\n",
-    "}"
+    "set_api_urls()\n",
```

Comment:
and same wherever else this `Set url and access token env vars` block appears

Reply:
I actually did not, intentionally, because the curators should never be messing around with dev or staging. And I don't want to confused them ("wait... what environment am I using? What is 'prod' exactly?").